File: ws/villagomez/u2/ws21zooWithInheritance/ws21Zoo/src/main/java/ec/espe/ws21zoo/model/Monkey.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.espe.ws21zoo.model;

import java.util.Date;

/**
 *
 * @author Domenica Villagomez, CommitCrew, DCCO-ESPE
 */
public class Monkey extends Primate{

    public Monkey(int id, String description, Date bornOnDate, char gender, int numberOfBones, Cage cage) {
        super(id, description, bornOnDate, gender, cage, numberOfBones);
    }
    
      public void feed(int quantity, Food food){
        System.out.println("Feeding the monkey with " + quantity + " of " + food);
    } 
}

File: hw/marquez/u2/HW18/HW18ZooInPythonJavaScript/zoo/ec.edu.espe.zoo.model/Monkey.js
const {Primate} = require('./Primate.js');

class Monkey extends Primate {
    constructor(id, name, birthDate, gender, numberOfLegs, cage) {
        super(id, name, birthDate, gender, numberOfLegs, cage);
    }

    Feed(quantity) {
        console.log(`Feeding the monkey with ${quantity} units of food.`);
    }
}

module.exports = { Monkey };
